### ISSUE TITLE
enable AfricaGrid VO

### ISSUE THREAD

**Issue by bazinski** [AAROC/chpctier2#8]
On sagrid queue
config is here : https://voms.ct.infn.it:8443/voms/vo.africa-grid.org/configuration/configuration.action

**Comment by brucellino** [comment]
If you enable this VO, we'll be able to accept jobs coming from most of the science gateways.

**Comment by bazinski** [comment]
True, once i have finished testing fmd and can prove the cvmfs is working happily on those 28 nodes, can open it up for abuse.

As an aside re the fmd calcs I see that that 1344 cores available to grid is > and a std user can get on the big cluster here.

**Comment by brucellino** [comment]
@bazinski how we looking on an ETA here ?
